src/gql/mutations/index.ts

```diff
@@ -1,9 +1,11 @@
 import ABORT_TASK from "./abort-task.graphql";
 import ADD_ANNOTATION from "./add-annotation.graphql";
 import ADD_FAVORITE_PROJECT from "./add-favorite-project.graphql";
+import ATTACH_PROJECT_TO_REPO from "./attach-project-to-repo.graphql";
 import ATTACH_VOLUME from "./attach-volume.graphql";
 import CLEAR_MY_SUBSCRIPTIONS from "./clear-my-subscriptions.graphql";
 import CREATE_PUBLIC_KEY from "./create-public-key.graphql";
+import DETACH_PROJECT_FROM_REPO from "./detach-project-from-repo.graphql";
```

Comment:
I noticed that after a project is detached from the repo, the "Save" button on the page becomes enabled. I'm guessing that that is not desired behavior?

Reply:
Right, I think all of these save issues can be saved for EVG-16268 because they stem from the same problem (the exception being if save is enabled when there is an error on the page—that kind of validation should be addressed now).

Reply:
makes sense, thanks!